vendor/src/github.com/docker/libnetwork/drivers/bridge/setup_ipv4.go

```diff
@@ -11,11 +11,21 @@ import (
 )
 
 func setupBridgeIPv4(config *networkConfiguration, i *bridgeInterface) error {
-	addrv4, _, err := i.addresses()
+	addrv4, addrv4List, _, err := i.addresses()
 	if err != nil {
 		return fmt.Errorf("failed to retrieve bridge interface addresses: %v", err)
 	}
 
+	// Iterate through all IPv4 addresses in case multiple IPv4 addresses exist
+	if config.AddressIPv4 != nil {
```

Comment:
Thanks @cpuguy83. The PR has been updated with the logic wrapped into a new func.